app/src/main/res/layout/fragment_route_list.xml

```diff
@@ -9,6 +9,7 @@
     <ListView
         android:id="@+id/list"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content" />
```

Comment:
We could fix this in activity_main.xml (this way it works for all fragments that are displayed)

You could try wrapping the Fragment and the BottomNavigationView with a relative Layout. Feel free to ask if you need any help ;)

Reply:
agree :)